examples/node-prerecorded/index.js

```diff
@@ -4,31 +4,32 @@ const fs = require("fs");
 const transcribeUrl = async () => {
   const deepgram = createClient(process.env.DEEPGRAM_API_KEY);
 
+  console.log("Transcribing URL", "https://dpgr.am/spacewalk.wav");
   const { result, error } = await deepgram.listen.prerecorded.transcribeUrl(
     {
       url: "https://dpgr.am/spacewalk.wav",
     },
     {
-      model: "nova",
+      model: "nova-2",
     }
   );
 
-  if (error) throw error;
-  if (!error) console.dir(result, { depth: null });
+  if (error) console.error(error);
+  if (!error) console.dir(result, { depth: 1 });
 };
 
 const transcribeFile = async () => {
   const deepgram = createClient(process.env.DEEPGRAM_API_KEY);
 
-  const { result, error } = await deepgram.listen.prerecorded.transcribeFile(
-    fs.readFileSync("./examples/nasa.mp4"),
-    {
-      model: "nova",
-    }
-  );
+  const file = fs.readFileSync("./examples/spacewalk.wav");
```

Comment:
```suggestion
 const filePath = path.join(__dirname, '../spacewalk.wav');
const file = fs.readFileSync(filePath);
```

Reply:
If you choose to add that code, would also need to add the path module:
```js
const path = require('path');
```

Reply:
Thanks, I'll use this for local files then